Azure/sonic-buildimage
# [as9716-32d]Fix thermal plan bug

**- Why I did it**
Fix as9716-32d thermal plan bug.

**- How I did it**
Add state=MAX when fan is plugged out. 
Plug out one fan , other fan speed will change to 100%.
But plug in the fan back , all fan speed still are 100%.
**- How to verify it**
Run thermal plan service, plug out fan-6.
Check fan speed=100%. Insert back fan-6, and check fan speed should according to thermal plan.
**- Which release branch to backport (provide reason below if selected)**




**- Description for the changelog**



**- A picture of a cute animal (not mandatory but encouraged)**
